js/index.js

```diff
@@ -0,0 +1,36 @@
+
+const menuElement = document.getElementById('navigation-elements');
+const menuButton = document.getElementById('menu-button');
+const logo =document.getElementById('logo');
+var page = document.getElementsByTagName('body')[0];
+
+
+
+function togglerMenu(){
+    if (menuButton.classList.contains('open-menu')) {
+        //menuElement.style.opacity = 1;
+        menuElement.classList.remove('hide-menu');
+        menuElement.classList.add('show-menu');
+        menuButton.classList.remove('open-menu');
+        menuButton.classList.add('close-menu');
+        logo.classList.add('text-logo-hidden');
+        page.classList.add('noscroll');
+    } else if(menuButton.classList.contains('close-menu')) {
+        //menuElement.style.opacity = 0;
+        menuElement.classList.remove('show-menu');
+        menuElement.classList.add('hide-menu');
+        menuButton.classList.remove('close-menu');
+        menuButton.classList.add('open-menu');
+        logo.classList.remove('text-logo-hidden');
+        page.classList.remove('noscroll');
+    }
```

Comment:
- [ ] [OPTIONAL] it would be better if you gather all the style in two classes and use them to toggle them when open and close